cli/register/register.go

```diff
@@ -279,11 +282,17 @@ func DoIt(org, pattern, nodeIdTok, userPw, email, inputFile string, nodeOrgFromF
 	//nd := Node{Id: nodeId, Token: nodeToken, Org: org, Pattern: pattern, Name: nodeId, HA: false}
 	falseVal := false
 	nd := api.HorizonDevice{Id: &nodeId, Token: &nodeToken, Org: &org, Pattern: &pattern, Name: &nodeName, HA: &falseVal} //todo: support HA config
-	httpCode, _ = cliutils.HorizonPutPost(http.MethodPost, "node", []int{201, 200, cliutils.ANAX_ALREADY_CONFIGURED}, nd)
-	if httpCode == cliutils.ANAX_ALREADY_CONFIGURED {
-		// Note: I wanted to make `hzn register` idempotent, but the anax api doesn't support changing existing settings once in configuring state (to maintain internal consistency).
-		//		And i can't query ALL the existing settings to make sure they are what we were going to set, because i can't query the node token.
-		cliutils.Fatal(cliutils.HTTP_ERROR, msgPrinter.Sprintf("this Horizon node is already registered or in the process of being registered. If you want to register it differently, run 'hzn unregister' first."))
+	// httpCode, _ = cliutils.HorizonPutPost(http.MethodPost, "node", []int{201, 200, cliutils.ANAX_ALREADY_CONFIGURED}, nd)
+	// if httpCode == cliutils.ANAX_ALREADY_CONFIGURED {
+	// 	// Note: I wanted to make `hzn register` idempotent, but the anax api doesn't support changing existing settings once in configuring state (to maintain internal consistency).
+	// 	//		And i can't query ALL the existing settings to make sure they are what we were going to set, because i can't query the node token.
+	// 	cliutils.Fatal(cliutils.HTTP_ERROR, msgPrinter.Sprintf("this Horizon node is already registered or in the process of being registered. If you want to register it differently, run 'hzn unregister' first."))
+	// }
+	err := CreateNode(nd, 60)
```

Comment:
Could you make the waiting time configurable in the cli/cliconfig/config.go? Default can be 60.  In large systems, each call to the exchange could take very long time.  You can just use one waiting time for all the functions below.

Reply:
Added